### PR TITLE
Temporary Christmas card page

### PR DESCRIPTION
**Pull request by Rhysimus**
All in one directory, to be used temporarily as a replacement for the cssbham.com/christmas/ url. It contains all the necessary files (images, HTML, CSS, JS) including a noscript version of the main thing which index.html redirects to if it identifies that JS is disabled.

This will work on the basis of how GitHub Pages handles files and doesn't use any Jekyll melarky. Let me know if it works as intended.

### CODE REVIEW

File: christmas/script.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+var isFirefox = typeof InstallTrigger !== 'undefined';
+
+$(function(){
+
+if (!$('.envelope').hasClass('open')){
+$('.envelope').click(function(){
```
Comment (LikkanChung) [inline christmas/script.js:6]: Indent these blocks

File: christmas/script.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    document.body.style.backgroundImage = "url('2.png')";
+});
+}
+
+});
+
+function resize() {
+    if(window.innerWidth / window.innerHeight >= 1.5) //Landscape
+    {
+    if(isFirefox)
```
Comment (LikkanChung) [inline christmas/script.js:17]: Also indent these for readability

File: christmas/styles.css [hunk 3]
```diff
@@ -0,0 +1,342 @@
+@import url(https://fonts.googleapis.com/css?family=Allan:700,400);
```
Comment (LikkanChung) [inline christmas/styles.css:1]: Also this file, adding indents

**Review by LikkanChung — CHANGES_REQUESTED** [review]
Adding some indents makes this code more readable, and easier to understand what's happening

File: christmas/index.html [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+        <div class="front">
+            <div class="mailme">
+            <p>A special message from CSS</p>
+            <p>(click!)</p>
+            </div>
+        </div>
+        <div class="back">
+            <div class="letter">
+              <img src="chrimsas.jpg" alt="" style="height: 75%; border-radius: 5px; position: absolute; left: 5px; top: 45px;">
+              <div style="position: absolute; right: 10px; top: 50px; width: 46%; text-align: center;">
```
Comment (LikkanChung) [inline christmas/index.html:26]: This is using inline styling. Why not extract it to the CSS import?
Also the left of the image and right of the text is different at 5 and 10px.

File: christmas/styles.css [hunk 5]
```diff
@@ -0,0 +1,344 @@
...
+        z-index: -1;
+    }
+}
+@keyframes letter-out {
+    0% {
+        transform: translate3d(0, 0, 0);
+        box-shadow: inset 0 0px 30px -5px #b08c5b, 0 0 10px -5px;
+    }
+    100% {
+        transform: translate3d(0, -733.33333px, 0);
```
Comment (LikkanChung) [inline christmas/styles.css:47]: In mobile devices the envelope is still partially visible. Is this intentional? 
image

File: christmas/noscript.html [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+    <div class="envelope open">
+        <div class="front">
+            <div class="mailme">
+            <p>A special message from CSS</p>
+            </div>
+        </div>
+        <div class="back">
+            <div class="letter">
+              <img src="chrimsas.jpg" alt="" style="height: 75%; border-radius: 5px; position: absolute; left: 5px; top: 45px;">
+              <div style="position: absolute; right: 10px; top: 50px; width: 46%; text-align: center;">
```
Comment (LikkanChung) [inline christmas/noscript.html:20]: Inline CSS. Extract to CSS file?

Also on mobile: 
image

Also doesn't properly resize on normal browser window, making text hard to read.

File: christmas/index.html [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+                <h1>Hello World!</h1>
+                <p>We are wishing you a very merry Christmas, and we hope you have so much fun!</p>
+                <p>From all the committee at CSS ❤️</p>
```
Comment (LikkanChung) [inline christmas/index.html:29]: This text is also quite hard to read on mobile devices without zooming in.

File: christmas/index.html (hunk 4 above)
Comment (Rhysimus) [reply christmas/index.html:26]: Fixed.

File: christmas/styles.css (hunk 5 above)
Comment (Rhysimus) [reply christmas/styles.css:47]: This is intentional 👍

File: christmas/noscript.html (hunk 6 above)
Comment (Rhysimus) [reply christmas/noscript.html:20]: noscript.html has no scaling as the approach relies on zoom and transform: scale (for Firefox only).

File: christmas/index.html (hunk 7 above)
Comment (Rhysimus) [reply christmas/index.html:29]: To stick with the scaling, this is intentional.

**Review by LikkanChung — APPROVED** [review]

**Pull request merged**